src/Symfony/Component/Validator/Constraints/UniqueValidator.php

```diff
@@ -21,6 +25,13 @@
  */
 class UniqueValidator extends ConstraintValidator
 {
+    private ?PropertyAccessorInterface $propertyAccessor;
+
+    public function __construct(PropertyAccessorInterface $propertyAccessor = null)
```

Comment:
I'd configure `factor` for this method and onInvalid set null, otherwise it does not make any sense because PropertyAccess does not public constructor and only way to create Accessor is to call static method `createPropertyAccessor`